src/Platform/Hosted/Components/ConnectExplorer.tsx

```diff
@@ -53,12 +66,18 @@ export const ConnectExplorer: React.FunctionComponent<Props> = ({
               id="connectWithConnectionString"
               onSubmit={async (event) => {
                 event.preventDefault();
+                setErrorMessage("");
 
                 if (isResourceTokenConnectionString(connectionString)) {
```

Comment:
we should probably also block resource tokens. These connection strings should also work to on if account is restricted